src/control/cmd/dmg/pretty/pool.go

```diff
@@ -29,6 +29,10 @@ func PrintPoolQueryResponse(pqr *control.PoolQueryResp, out io.Writer, opts ...P
 	// Maintain output compatibility with the `daos pool query` output.
 	fmt.Fprintf(w, "Pool %s, ntarget=%d, disabled=%d, leader=%d, version=%d\n",
 		pqr.UUID, pqr.TotalTargets, pqr.DisabledTargets, pqr.Leader, pqr.Version)
+	if pqr.PoolLayoutVer != pqr.UpgradeLayoutVer {
```

Comment:
Could it be possible that pqr.PoolLayoutVer > pqr.UpgradeLayoutVer. In this case, the error message should probably not the same.

Reply:
We don't support downgrading case, so pool service won't up in this case, so it will not happen.

Reply:
OK, thanks for the info.